lib/TPP/PassBundles/LowLevelParallelization.cpp

```diff
@@ -65,11 +65,5 @@ struct LowLevelParallelization
     mlir::tpp::SCFParallelLoopTilingOptions tilingOptions;
     tilingOptions.tileSizes = parallelTaskGrid;
     pm.addPass(createSCFParallelLoopTiling(tilingOptions));
-
-    pm.addNestedPass<func::FuncOp>(createIntelAMXTileConfigInsertionPass());
-    pm.addNestedPass<func::FuncOp>(createCanonicalizerPass());
-    pm.addNestedPass<func::FuncOp>(createLoopInvariantCodeMotionPass());
```

Comment:
We need to avoid moving passes out of bundles into the pipeline itself. Perhaps this is just an artifact of the duality xsmm/vectorization but it needs understanding if this is the best way through this.